packages/analytics/analytics-chart/src/components/chart-types/TimeSeriesChart.vue

```diff
@@ -139,15 +141,26 @@ const props = defineProps({
     required: false,
     default: (a: TooltipEntry, b: TooltipEntry) => b.rawValue - a.rawValue,
   },
+  zoom: {
+    type: Boolean,
+    required: false,
+    default: false,
+  },
 })
 
+const emit = defineEmits<{
+  (e: 'zoom-time-range', newTimeRange: AbsoluteTimeRangeV4): void,
+}>()
+
 const { translateUnit } = composables.useTranslatedUnits()
 const chartInstance = ref<{ chart: Chart }>()
 const legendID = ref(uuidv4())
 const chartID = ref(uuidv4())
 const legendItems = ref<LegendItem[]>([])
 const tooltipElement = ref()
 const legendPosition = ref(inject('legendPosition', ChartLegendPosition.Right))
+const isDoingSelection = ref(false)
+const tz = (new Intl.DateTimeFormat()).resolvedOptions().timeZone
```

Comment:
I _think_ it's better to not plug the timezone in at this point.  (It should be an optional member of `AbsoluteTimeRangeV4`.)